galicaster/plugins/camera_control.py

```diff
@@ -0,0 +1,252 @@
+# -*- coding:utf-8 -*-
+# Galicaster, Multistream Recorder and Player
+#
+#       galicaster/plugins/camera_control
+#
+# Copyright (c) 2016, Teltek Video Research <galicaster@teltek.es>
+#
+# This work is licensed under the Creative Commons Attribution-
+# NonCommercial-ShareAlike 3.0 Unported License. To view a copy of
+# this license, visit http://creativecommons.org/licenses/by-nc-sa/3.0/
+# or send a letter to Creative Commons, 171 Second Street, Suite 300,
+# San Francisco, California, 94105, USA.
+
+import threading
+import Queue
+
+from galicaster.core import context
+from galicaster.core.core import PAGES
+from galicaster.classui import get_ui_path, get_image_path
+
+import galicaster.utils.pysca as pysca
+
+from gi.repository import Gtk, Gdk, GObject, Pango, GdkPixbuf
+
+class T(threading.Thread):
```

Comment:
Good Idea.

It is the same class that is defined into the worker. Refactor the class in the utils module.